service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/support/MigrateCaseController.java

```diff
@@ -103,6 +103,11 @@ private void run2961(CaseDetails caseDetails) {
                 caseDetails.getId()));
         }
 
+        if (caseData.getApplicants() == null) {
```

Comment:
do you to check on the possibility of it being empty as well?

Reply:
Likely to close this PR and filter out in the migration tool. We can then update the query to silently log cases with no applicants.